docs/api.rst

```diff
@@ -3422,6 +3422,12 @@ AuditLogDiff
 
         :type: :class:`str`
 
+    .. attribute:: guild
+
+        The guild of something.
+
+        :type: :class:`Guild`
+
```

Comment:
I can't imagine a scenario where this comes up right now, but this string leaves much to be desired.

Reply:
It's in the `TRANSFORMERS` in `AuditLogChanges`:
https://github.com/Rapptz/discord.py/blob/cb6170a7247958294b85a3bd0ab29b71088e86a2/discord/audit_logs.py#L353

But I don't know right now where that would be needed either. So I simply took "[...] of something", as it is also the case with `name`. Should I remove it from the docs accordingly?

Reply:
I guess it's fine, though I'll think of something for the replacement string.